server/scripts/2020TaxSeasonDataScript.php

```diff
@@ -302,6 +305,122 @@ function insertSoutheastCommunityCollegeData() {
 	}
 }
 
+function insertInternationalStudentScholarSiteData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+	$reviewerRoleId = 4;
+	
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The ISS data has already been inserted');
+	}
+
+	$siteId = 4; // Manually obtained from PROD DB
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		// Tuesdays
+		$tuesdayDates = array('2020-03-03', '2020-03-10', '2020-03-17', '2020-03-31');
+		foreach ($tuesdayDates as $date) {
+			$shiftId = insertShift("$date 13:00:00", "$date 18:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 41, 41, 100, 60, $siteId);

```

Comment:
Not sure it's an issue--but technically we don't need a min/max appointments on these since it's hard-coded in the back-end rn unfortunately. So those values aren't used, but I think they can stay